packages/client-core/src/user/menus/ProfileMenu.tsx

```diff
@@ -703,34 +702,6 @@ const ProfileMenu = ({ hideLogin, onClose }: Props): JSX.Element => {
             <LogIn01Lg />
             {t('user:usermenu.profile.logout.submit')}
           </button>
-
-          <button
```

Comment:
I also see a `packages/client-core/src/components/Settings/DeleteAccountScreen.tsx` control which can delete a user. Where is it used? It still introduces the loop hole to cause the original issue.

We can get rid of `AuthService.removeUser` method if above question is sorted out.